drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -951,5 +979,22 @@ function DragAndDropBlock(runtime, element, configuration) {
         });
     };
 
+    /**
+     * markItemZoneAlign: Mark the items placed in an aligned zone with the zone
+     * alignment, so they can be properly placed inside the zone.
+     * We have do this in JS, not python, since zone configurations may change.
+     */
+    var markItemZoneAlign = function() {
+        var zone_align = {};
```

Comment:
@pomegranited Could you rename this to `zone_alignments`? That would make it easier to distinguish it from `item.zone_align`.